examples/CustomPresentationExample.swift

```diff
@@ -99,6 +99,24 @@ final class VerticalSheetTransition: NSObject, Transition {
 
     CATransaction.commit()
   }
+  
+  //uncomment to make animation interactive
+  /*
+  var i = 0.01
+  func start(withInteractiveContext context: InteractiveTransitionContext) {
+    let timer = DispatchSource.makeTimerSource(queue: DispatchQueue.main)
+    timer.scheduleRepeating(deadline: .now(), interval: .milliseconds(10), leeway: .milliseconds(10))
+    timer.setEventHandler {
+      self.i += 0.01
+      context.updatePercent(CGFloat(self.i))
+      if(self.i > 1.0) {
+        context.finishInteractiveTransition()
+        timer.cancel()
+      }
+    }
+    timer.resume()
+  }
+   */
```

Comment:
Create a new example and include this timer code there so that users can play with the demo more easily.